### PR TITLE
Added quiz and result page

### PR DESCRIPTION
**Pull request by parthgabani777**
- Added quiz and result page.
- Created service for fetching questions.

### CODE REVIEW

File: src/pages/quiz/quiz.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+                        />
+
+                        {/* <div className="question-footer">
+                            <Link to="/result" className="submit-btn">
+                                End test
+                            </Link>
+                        </div> */}
+                    </div>
+                </>
+            ) : (
```
Comment (Manpreet-hub) [inline src/pages/quiz/quiz.js:106]: remove commented code if not required.

**Review by Manpreet-hub — APPROVED** [review]
The code looks good !

File: src/pages/quiz/quiz.js [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+                <>
+                    <div className="quiz-title">{quiz.title}</div>
+
+                    <div className="question-container">
+                        <div className="question-header">
+                            <p className="question-number">
+                                <span className="question-number-text">
+                                    Question Index:{" "}
+                                </span>
+                                {answers.currentQuestionCounter}/5
```
Comment (ashishnnnnn) [inline src/pages/quiz/quiz.js]: 5 should replaced by questions.length because i found in html quiz there are only 3 questions..

**Review by ashishnnnnn — APPROVED** [review]
Code seems pretty much clean .. apart from the suggestion..

File: src/pages/quiz/quiz.js (hunk 2 above)
Comment (parthgabani777) [reply src/pages/quiz/quiz.js]: Done. Thank you for pointing out I forgot it.

File: src/pages/quiz/quiz.js (hunk 1 above)
Comment (parthgabani777) [reply src/pages/quiz/quiz.js:106]: Done.

**Pull request merged**